Formula/chkrootkit.rb

```diff
@@ -1,9 +1,8 @@
 class Chkrootkit < Formula
   desc "Rootkit detector"
   homepage "http://www.chkrootkit.org/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
```

Comment:
No problem about the wait.

You're right about being able to squash from the GitHub page, but formulae with bottles need to be merged off-site using `brew pull --bottle` to ensure that the binaries get updated as well. I was still in the wrong, though; I was being lazy and didn't take into account your time or resource constraints. My apologies!